subt/simulation.py

```diff
@@ -0,0 +1,159 @@
+import collections
+import datetime
+import inspect
+import math
+
+from threading import Thread, RLock
+
+from osgar.lib import quaternion
+
+try:
+    from colored import stylize, fg
```

Comment:
How important is this coloring? There seems to be lots of boilerplate and non-obvious code related to it.

Reply:
It made the output much more readable but I agree that what it added to readability of the output, it removed from the readability of the actual code. Some day I'd like to have something like it, but probably not today.